src/main/java/com/ftpix/sherdogparser/models/Fighter.java

```diff
@@ -10,7 +10,15 @@
     private String weight = "";
     private Date birthday;
     private int wins = 0;
+    private int winsKO = 0;
```

Comment:
We need to follow camelCase writing for variable names.
i.e.: winsKo, winsSub, winsDec, winsOthers.